### PR TITLE
Remove generated code for unused classes. CommunicationErrorResponse, OfferingsResponse, PhoneNumberAreaCodes, PhoneNumberCountries, PhoneNumberLocalities, PurchasedPhoneNumbers.

### PR DESCRIPTION
**Pull request by whisper6284**
# Contributing to the Azure SDK

Remove generated code for unused classes. CommunicationErrorResponse, OfferingsResponse, PhoneNumberAreaCodes, PhoneNumberCountries, PhoneNumberLocalities, PurchasedPhoneNumbers.

This will reduce redundancy in the Azure SDK and provide a more accurate code coverage number.

### CODE REVIEW

File: sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumberOperationType.cs [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
 using Azure.Core;
 
 namespace Azure.Communication.PhoneNumbers
 {
     [CodeGenModel("PhoneNumberOperationType")]
-    internal readonly partial struct PhoneNumberOperationType
+    /* [CodeGenSuppress("PhoneNumberOperationType", typeof(string))]
```
Comment (sorrego-msft) [inline sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumberOperationType.cs:10]: Is this commented code needed?

File: sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumberOperationStatus.cs [hunk 2]
```diff
@@ -6,6 +6,7 @@
 namespace Azure.Communication.PhoneNumbers
 {
     [CodeGenModel("PhoneNumberOperationStatus")]
+    [CodeGenSuppress("PhoneNumberOperationStatus")]
```
Comment (sorrego-msft) [inline sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumberOperationStatus.cs:9]: Is there a difference between OperationStatus and OperationType? 
Status is internal readonly and has the CodeGenSuppress attribute while Type was updated to be public and does not have the codegensuppress attribute.

**Review by sorrego-msft — APPROVED** [review]

**Review by danielortega-msft — COMMENTED** [review]
While the changes themselves look good, I'm worried about the fact that it appears to have created multiple public types, at least based on the API diff generated. Considering that all of these were generated classes and not our public models, shouldn't they have been kept internal?

**Comment by whisper6284** [comment]
> While the changes themselves look good, I'm worried about the fact that it appears to have created multiple public types, at least based on the API diff generated. Considering that all of these were generated classes and not our public models, shouldn't they have been kept internal?

Reverting these changes. These were accidentally included.

**Pull request merged**